apiserver/facades/client/application/charmstore.go

```diff
@@ -206,6 +206,9 @@ type CharmArchive struct {
 
 	// Macaroon is the authorization macaroon for accessing the charmstore.
 	Macaroon macaroon.Slice
+
+	// Charm Version
```

Comment:
We should add a little more info, and end with a "."
eg
// CharmVersion contains the semantic version of the charm, typically the output of git describe.